lib/pages/setup/route_list.dart

```diff
@@ -53,89 +54,167 @@ class _RouteListState extends State<RouteList> {
-    text += LocalizationService().getLocalization(english: "Altitude:", german: "Höhe:") + '${widget.routeParams.altitudeType}\n';
-    print(text);
-    summary = text;
-  }
-
   @override
   Widget build(BuildContext context) {
     Widget body;
     if(_routesCalculated) {
-      body = Stack(children: <Widget>[
-          Text(
-            summary,
-            style: TextStyle(
-                fontSize: 18,
-                fontWeight: FontWeight.bold,
-                color: Colors.grey[600]),
-            textAlign: TextAlign.left,
-          ),
-          Padding(padding: EdgeInsets.only(top: 20)),
-          ListView.builder(
-            itemCount: routeList.length,
+      body = 
+        Column(
+        children: <Widget>[
+        Padding(padding: EdgeInsets.only(top: 4)),
+          headerText(),
+        Padding(padding: EdgeInsets.only(top: 4)),
+        Expanded(
+          child: ListView.builder(
+            itemCount: _routeList.length,
             itemBuilder: (context, index) {
               return Padding(
                 padding:
-                    const EdgeInsets.symmetric(vertical: 1.0, horizontal: 4.0),
+                  const EdgeInsets.symmetric(vertical: 1.0, horizontal: 1.0),
                 child: Card(
                   child: ListTile(
                     onTap: () {
                       widget.routeParams.routeIndex = index;
                       widget.onPushRoutePreview(widget.routeParams);
                     },
-                    title: Text(routeList[index].title),
-                    subtitle: Text(
-                        LocalizationService().getLocalization(english: "Distance:", german: "Distanz:") + '${routeList[index].distance.toString()}\n${LocalizationService().getLocalization(english: "Date", german: "Datum")}: ${routeList[index].date}'),
-                    leading: CircleAvatar(
-                        child: Icon(
-                      Icons.directions_walk,
-                      color: htwGreen,
-                    )
-                        //backgroundImage: (routeList[index].avatar == null) ? AssetImage('assets/img/h4n-icon2.png') : AssetImage('assets/img/h4n-icon2.png'),
-                        ),
+                    subtitle: buildSub(_routeList[index]),
+                    isThreeLine: true,
                   ),
                 ),
               );
             },
           ),
-        ]);
+        )]);
     }
     else {
       body = Center(
         child: new CircularProgressIndicator(),
       );
     }
+
     return Scaffold(
         backgroundColor: Colors.grey[200],
         appBar: AppBar(
           backgroundColor: htwGreen,
-          title: Text(LocalizationService().getLocalization(english: "Choose a route to preview", german: "Route für Vorschau wählen")),
+          title: Text(LocalizationService().getLocalization(english: 'Choose a route to preview', german: 'Route für Vorschau wählen')),
```

Comment:
why changing the " to ' though?

Reply:
Because I use only ' and it looks odd and inconsistent  using both